hierarchy/store.go

```diff
@@ -229,9 +228,83 @@ func setHasData(connection bolt.Conn, instanceID, dimensionName string) error {
 	defer stmt.Close()
 
 	_, err = stmt.ExecNeo(nil)
+	return err
+}
+
+func markNodesToRemain(connection bolt.Conn, instanceID, dimensionName string) error {
```

Comment:
Is it possible to add unit tests for some of those methods?

Reply:
Added tests for all of the store methods 👍